sections/package.zsh

```diff
@@ -23,20 +24,25 @@ spaceship_package() {
   [[ $SPACESHIP_PACKAGE_SHOW == false ]] && return
 
   # Show package version only when repository is a package
-  # @todo: add more package managers
-  [[ -f package.json ]] || return
-
-  spaceship::exists npm || return
-
   local 'package_version'
 
-  if spaceship::exists jq; then
-    package_version=$(jq -r '.version' package.json 2>/dev/null)
-  elif spaceship::exists python; then
-    package_version=$(python -c "import json; print(json.load(open('package.json'))['version'])" 2>/dev/null)
-  elif spaceship::exists node; then
-    package_version=$(node -p "require('./package.json').version" 2> /dev/null)
-  fi
+  [[ -f package.json ]] && spaceship::exists npm && {
```

Comment:
Could you split it up as it was before? Plain `if`s look more readable and understandable for newbies who come to contribute to spaceship for the first time?